internal/cli/email_templates.go

```diff
@@ -192,12 +192,30 @@ func updateEmailTemplateCmd(cli *cli) *cobra.Command {
 			}
 
 			var oldTemplate *management.EmailTemplate
+			templateExists := true
 			err := ansi.Waiting(func() (err error) {
 				oldTemplate, err = cli.api.EmailTemplate.Read(apiEmailTemplateFor(inputs.Template))
 				return err
 			})
 			if err != nil {
-				return fmt.Errorf("failed to get the email template '%s': %w", inputs.Template, err)
+				wrappedError := fmt.Errorf("failed to get the email template '%s': %w", inputs.Template, err)
+
+				mErr, ok := err.(management.Error)
+				if !ok {
```

Comment:
Could this be `if !ok || mErr.Status() != http.StatusNotFound`?

Reply:
Certainly 👍🏻 [`ad1086e` (#611)](https://github.com/auth0/auth0-cli/pull/611/commits/ad1086eb87d633b6dce8352acf8b045624b2c281)